### PR TITLE
EZP-28289: Updated UI colors - Content view

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-28289](https://jira.ez.no/browse/EZP-28289)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Most important aspects:
- Updated background-color of content view views, according to design specifications;
- Tested & updated navbar link color, background-color and sub-navbar border-bottom;
- Reviewed nav-tabs and harmonized active tab content background-color;
- Updated `.ez-field-group` header background-color (and spacing here) based on design specifications.

screen shot 2017-12-06 at 4 02 28 pm
screen shot 2017-12-06 at 4 02 55 pm
screen shot 2017-12-06 at 4 03 32 pm

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/admin/content_type/view.html.twig [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'content_type' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (dew326) [inline src/bundle/Resources/views/admin/content_type/view.html.twig]: This `bodyClass` should reflect where you are in the admin panel (this is done to have the context in CSS).
`ez-content-view` means that you are in the `content structure`

File: src/bundle/Resources/public/scss/_main-nav.scss [hunk 2]
```diff
@@ -13,28 +13,28 @@
...
-        background-color: $ez-ground-base-light;
+        background-color: $ez-ground-base-medium;
         border: none;
         margin-bottom: 0;
 
         .nav-link {
             padding: 1rem 2rem;
-            color: $ez-color-base-dark;
+            color: $ez-black;
             display: flex;
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_main-nav.scss]: It makes me think that we're missing the `transition: all .3s <timing-function>;` property here, to make the color changes nicer for end user.

File: src/bundle/Resources/views/admin/content_type/view.html.twig (hunk 1 above)
Comment (sunpietro) [inline src/bundle/Resources/views/admin/content_type/view.html.twig]: It's `ez-content-type-view`

File: src/bundle/Resources/views/admin/content_type_group/list.html.twig [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'content_type' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/content_type_group/list.html.twig]: `ez-content-type-group-list-view`

File: src/bundle/Resources/views/admin/content_type_group/view.html.twig [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'content_type' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/content_type_group/view.html.twig]: `ez-content-type-group-view`

File: src/bundle/Resources/views/admin/language/list.html.twig [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'language' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/language/list.html.twig]: `ez-language-list-view`

File: src/bundle/Resources/views/admin/language/view.html.twig [hunk 6]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'language' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/language/view.html.twig]: `ez-language-view`

File: src/bundle/Resources/views/admin/role/list.html.twig [hunk 7]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'role' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/role/list.html.twig]: `ez-role-list-view`

File: src/bundle/Resources/views/admin/role/view.html.twig [hunk 8]
```diff
@@ -2,6 +2,8 @@
 
 {% trans_default_domain 'role' %}
 
+{% block bodyClass %}ez-info-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/role/view.html.twig]: `ez-role-view`

File: src/bundle/Resources/views/admin/search/search.html.twig [hunk 9]
```diff
@@ -4,6 +4,8 @@
 
 {% block pageTitle %}{% endblock %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/search/search.html.twig]: `ez-search-view`

File: src/bundle/Resources/views/admin/section/list.html.twig [hunk 10]
```diff
@@ -1,6 +1,8 @@
 {% extends 'EzPlatformAdminUiBundle::layout.html.twig' %}
 {% trans_default_domain 'section' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/section/list.html.twig]: `ez-section-list-view`

File: src/bundle/Resources/views/admin/section/view.html.twig [hunk 11]
```diff
@@ -4,6 +4,8 @@
 
 {% form_theme form_section_delete 'EzPlatformAdminUiBundle:parts/form:flat_widgets.html.twig' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/section/view.html.twig]: `ez-section-view`

File: src/bundle/Resources/views/admin/trash/list.html.twig [hunk 12]
```diff
@@ -6,6 +6,8 @@
 
 {% form_theme form_trash_item_restore '@EzPlatformAdminUi/form_fields.html.twig' %}
 
+{% block bodyClass %}ez-content-view{% endblock %}
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/trash/list.html.twig]: `ez-trash-list-view`

**Comment by inakijv** [comment]
Updated based on reviews

**Pull request merged**